docs/interactions/Slash_Commands.md

```diff
@@ -162,7 +162,7 @@ Full documentation of endpoints can be found in [Endpoints](#DOCS_INTERACTIONS_S
 
 When a user uses a Slash Command, your app will receive an **Interaction**. Your app can receive an interaction in one of two ways:
 
-- Via gateway event, `INTERACTION_CREATE` <docs>
+- Via gateway event, [INTERACTION_CREATE](#DOCS_TOPICS_GATEWAY/interaction-create)
```

Comment:
```suggestion
- Via gateway event, [Interaction Create](#DOCS_TOPICS_GATEWAY/interaction-create)
```
Events are written like that throughout the documentation

Reply:
reworded the whole line, should match the rest of the docs